### PR TITLE
[WIP] Update api docs for PretrainedModel and PretrainedTokenizer

### PR DESCRIPTION
**Pull request by guoshengCS**
### PR types

Others
### PR changes

Docs
### Describe

Update api docs for PretrainedModel.

### CODE REVIEW

File: paddlenlp/transformers/model_utils.py [hunk 1]
```diff
@@ -54,24 +68,40 @@ def register_base_model(cls):
...
-    - `pretrained_resource_files_map` (dict): The dict has the same keys as
-      `resource_files_names`, the values are also dict mapping specific pretrained
-      model name to URL linking to pretrained model.
-    - `pretrained_init_configuration` (dict): The dict has pretrained model names
-      as keys, and the values are also dict preserving corresponding configuration
-      for model initialization.
-    
-    - `base_model_prefix` (str): represents the the attribute associated to the
+    The base class for all pretrained models. It mainly provides common methods
+    for pretrained models loading (construction and loading) and saving. Loading
```
Comment (yingyibiao) [inline paddlenlp/transformers/model_utils.py:72]: for pretrained models loading (construction and loading) and saving
->
for loading (construction and loading) and saving pretrained models

File: paddlenlp/transformers/model_utils.py [hunk 2]
```diff
@@ -54,24 +68,40 @@ def register_base_model(cls):
...
+    and saving also rely on the following class attributes which should be overridden
+    by derived classes accordingly:
+
+    - `model_config_file` (`str`): Represents the file name of model configuration
```
Comment (yingyibiao) [inline paddlenlp/transformers/model_utils.py:76]: 这里的变量名感觉加粗比较合适，下同：
`model_config_file` -> **model_config_file**

File: paddlenlp/transformers/model_utils.py [hunk 3]
```diff
@@ -54,24 +68,40 @@ def register_base_model(cls):
...
+      for configuration saving and loading in local file system. The value is
```
Comment (yingyibiao) [inline paddlenlp/transformers/model_utils.py:77]: Represents the file name of model configuration for configuration saving and loading in local file system.
->
Name of local file where the model configuration can be saved and loaded locally.

File: paddlenlp/transformers/model_utils.py [hunk 4]
```diff
@@ -54,24 +68,40 @@ def register_base_model(cls):
...
+      `model_config.json`.
+    - `resource_files_names` (`dict`): Represents resources to specific file
+      names mapping for resource saving and loading in local file system. Currently,
+      resources only include the model state, thus the dict only includes
+      `'model_state'` as key with corresponding value `'model_state.pdparams'`
+      for model weights saving and loading.
+    - `pretrained_init_configuration` (`dict`): Provides the model configurations
+      of built-in pretrained models (constract to models in local file system).
```
Comment (yingyibiao) [inline paddlenlp/transformers/model_utils.py:85]: typo, 下同：
constract -> contrasts

File: paddlenlp/transformers/model_utils.py [hunk 5]
```diff
@@ -54,24 +68,40 @@ def register_base_model(cls):
...
+      It has pretrained model names as keys (such as `bert-base-uncased`), and
```
Comment (yingyibiao) [inline paddlenlp/transformers/model_utils.py:86]: string应该加上引号，下同：
`bert-base-uncased` -> "bert-base-uncased"

File: paddlenlp/transformers/model_utils.py [hunk 6]
```diff
@@ -54,24 +68,40 @@ def register_base_model(cls):
...
+      the values are dict preserving corresponding configuration for model initialization.
+    - `pretrained_resource_files_map` (`dict`): Provides resource URLs of built-in
+      pretrained models (constract to models in local file system). It has the
+      same keys as `resource_files_names` (that is `'model_state'`), and the
+      values are also `dict` mapping specific pretrained model name (such as
+      `bert-base-uncased`) to URL of pretrained model weights.
```
Comment (yingyibiao) [inline paddlenlp/transformers/model_utils.py:92]: It has the same keys as `resource_files_names` (that is `'model_state'`), and the values are also `dict` mapping specific pretrained model name (such as `bert-base-uncased`) to URL of pretrained model weights.
->
It has the same key as `resource_files_names` (that is "model_state"), and the corresponding value is a dict with specific model name to model weights URL mapping (such as "bert-base-uncased" -> "https://paddlenlp.bj.bcebos.com/models/transformers/bert-base-uncased.pdparams").

File: paddlenlp/transformers/model_utils.py [hunk 7]
```diff
@@ -38,10 +38,24 @@
...
+    class. It indicates the base model class in derived classes of the same architecture.
+
     Args:
-        cls (class): the name of the model
+        cls (`type`): The `PretrainedModel` class to be decorated.
+    
+    Returns:
+        `type`: The decorated `PretrainedModel` class. It's same as the input
+            argument `cls`.
+
```
Comment (yingyibiao) [inline paddlenlp/transformers/model_utils.py:51]: 这一块rtd上显示有点问题

File: paddlenlp/transformers/model_utils.py [hunk 8]
```diff
@@ -111,20 +150,31 @@ def get_output_embeddings(self):
...
-            pretrained_model_name_or_path (str): A name of or a file path to a
-                pretrained model.
-            *args (tuple): position arguments for `__init__`. If provide, use
-                this as position argument values for model initialization.
-            **kwargs (dict): keyword arguments for `__init__`. If provide, use
-                this to update pre-defined keyword argument values for model
-                initialization. If the key is in base model `__init__`, update
-                keyword argument of base model; else update keyword argument of
-                derived model.
+            pretrained_model_name_or_path (`str`): A name of pretrained model
```
Comment (yingyibiao) [inline paddlenlp/transformers/model_utils.py:158]: A name -> Name

File: paddlenlp/transformers/model_utils.py [hunk 9]
```diff
@@ -111,20 +150,31 @@ def get_output_embeddings(self):
...
+                for built-in pretrained models loading, such as `bert-base-uncased`.
+                Or a local file directory path for local trained models loading.
+            *args (tuple): position arguments for model `__init__`. If provided,
```
Comment (yingyibiao) [inline paddlenlp/transformers/model_utils.py:161]: 注意大小写，下同
position -> Position

File: paddlenlp/transformers/model_utils.py [hunk 10]
```diff
@@ -111,20 +150,31 @@ def get_output_embeddings(self):
     @classmethod
     def from_pretrained(cls, pretrained_model_name_or_path, *args, **kwargs):
         """
-        Instantiate an instance of `PretrainedModel` from a predefined
-        model specified by name or path.
+        Create an instance of `PretrainedModel` and load pretrained model weights
```
Comment (yingyibiao) [inline paddlenlp/transformers/model_utils.py:153]: 注意统一动词单复数，下同
Create -> Creates

**Comment by yingyibiao** [comment]
todo: update Bert docstrings

**Comment by ZeyuChen** [comment]
@yingyibiao Fix conflicting files then merge code.

**Comment by yingyibiao** [comment]
LGTM

**Review by yingyibiao — APPROVED** [review]

**Pull request merged**